### PR TITLE
Added eraPoints, commission, controllerId, NextSessionIds

### PR DESCRIPTION
**Pull request by nblogist**
Conversation at https://github.com/hicommonwealth/chain-events/issues/27
I have added Erapoints, Commission, controllerId, NextSessionKeys in the new Session Event

### CODE REVIEW

**Comment by jnaviask** [comment]
Tests failing?

Also, this is a lot of overhead for every event. Do we want to be doing this? @drewstone

**Comment by drewstone** [comment]
@zohaib01khan are you taking this PR over? I think this is fine @jnaviask, these queries are only being run each hour.

**Comment by drewstone** [comment]
We can likely optimise all this querying though, but I would like to know what eventHandlers these larger querys like validatorInfo are going to be used for, maybe @nblogist @zohaib01khan can shed some light on the intention

File: src/substrate/types.ts [hunk 1]
```diff
@@ -181,6 +181,7 @@ export interface INewSession extends IEvent {
   waiting: Array<AccountId>;
   sessionIndex: number;
   currentEra?: number;
+  validatorInfo : {},
```
Comment (drewstone) [inline src/substrate/types.ts:184]: add type

**Review by drewstone — CHANGES_REQUESTED** [review]

**Comment by zohaib01khan** [comment]
> @zohaib01khan are you taking this PR over? I think this is fine @jnaviask, these queries are only being run each hour.

@drew new-session and rewards event handlers will be utilizing this information.
https://github.com/hicommonwealth/commonwealth/pull/782/files/efacb45e9cd3e0d0b0467ce48b4ea0eff4137a79#diff-984a722de7058e672a30357c64ffcf5eR89-R90

https://github.com/hicommonwealth/commonwealth/pull/782/files/efacb45e9cd3e0d0b0467ce48b4ea0eff4137a79#diff-0f1fe4ace565ee63a6626bdd411d58ccR62

File: src/substrate/types.ts (hunk 1 above)
Comment (nblogist) [reply src/substrate/types.ts]: Done

**Comment by nblogist** [comment]
@drewstone  Any updates on this?

**Comment by drewstone** [comment]
Why is the build failing with `unwrap is not a function` @nblogist ?

**Comment by nblogist** [comment]
@drewstone I tried without unwrap, unwrap was being called on an intermediate value (in a Promise), causing code to break.
@jnaviask  Tests are passing now.

Please let me know if there's anything else.

**Pull request merged**